crates/router/src/core/payments/transformers.rs

```diff
@@ -971,44 +971,57 @@ impl ForeignFrom<(storage::PaymentIntent, storage::PaymentAttempt)> for api::Pay
 }
 
 #[cfg(feature = "payouts")]
-impl ForeignFrom<(storage::Payouts, storage::PayoutAttempt)> for api::PayoutCreateResponse {
-    fn foreign_from(item: (storage::Payouts, storage::PayoutAttempt)) -> Self {
+impl ForeignFrom<(storage::Payouts, storage::PayoutAttempt, domain::Customer)>
+    for api::PayoutCreateResponse
+{
+    fn foreign_from(item: (storage::Payouts, storage::PayoutAttempt, domain::Customer)) -> Self {
         let payout = item.0;
         let payout_attempt = item.1;
+        let customer = item.2;
```

Comment:
can we do it something like 
```suggestion
        (payout, payout_attempt, customer) = item;
```